<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>professorwindow</class>
 <widget class="QDialog" name="professorwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>806</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Dashboard</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Routine</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>1111</width>
             <height>721</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Faculty</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="pw_faculty">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Year</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="pw_year">
                  <item>
                   <property name="text">
                    <string>1st</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2nd</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>3rd</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>4th</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Semester</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="pw_semester">
                  <item>
                   <property name="text">
                    <string>1st</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2nd</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Day</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="pw_day_2">
                  <item>
                   <property name="text">
                    <string>Sunday</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Monday</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Tuesday</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Wednesday</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Thursday</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Friday</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="pw_table"/>
            </item>
            <item>
             <widget class="QPushButton" name="pw_show_2">
              <property name="text">
               <string>Show</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Class Status</string>
          </attribute>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>20</y>
             <width>641</width>
             <height>681</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>500</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>11</number>
            </property>
            <property name="topMargin">
             <number>11</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Block</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="pw_block">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>40</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Room No.</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="pw_room">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>40</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Time</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="pw_start_time">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>40</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>6</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>7</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>8</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>9</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>10</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>11</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>12</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>13</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>14</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>15</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>16</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>17</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>18</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>65</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>to</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="pw_end_time">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>40</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>7</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>8</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>9</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>10</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>11</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>12</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>13</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>14</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>15</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>16</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>17</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>18</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>19</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Day</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="pw_day">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>40</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Sunday</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Monday</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Tuesday</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Wednesday</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Thursday</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Friday</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pw_show">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Show</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Request</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Faculty</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="req_faculty"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Year</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="req_year"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Semester</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="req_semester"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_17">
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Day</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="req_day"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Time</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="req_start_time"/>
               </item>
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>to</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="req_end_time"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Block</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="req_block"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Room</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="req_room"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Subject</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="req_subject"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="request_button">
               <property name="text">
                <string>Request</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
